app/src/main/java/com/limelight/Game.java

```diff
@@ -168,6 +168,12 @@ public void onServiceDisconnected(ComponentName componentName) {
     public static final String EXTRA_APP_HDR = "HDR";
     public static final String EXTRA_SERVER_CERT = "ServerCert";
 
+    @Override
+    public void onBackPressed() {
+        super.onBackPressed();
+        finish();
```

Comment:
This approach will break the back button on gamepads and mice.

The correct approach is to modify the `ControllerHandler.shouldIgnoreBack()` function here: https://github.com/moonlight-stream/moonlight-android/blob/3e0bf25acb900b99856047d8ab83861f7bb613cb/app/src/main/java/com/limelight/binding/input/ControllerHandler.java#L439

Ideally we wouldn't match on the specific device name since that can vary significantly based on device. We should perform some heuristic based on the InputDevice characteristics to determine whether the back button should work for navigation or in-game controls (logcat logs will help with that).